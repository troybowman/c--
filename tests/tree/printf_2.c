extern void printf(char format[]);

void main(void)
{
  printf("this doesn't work: %d\n", 5);
}
