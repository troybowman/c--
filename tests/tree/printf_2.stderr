error: expected 1 arguments for function printf, 2 were provided. line 5
c-- returned nonzero exit code